model/src/main/java/org/cbioportal/model/StructuralVariantIdentifier.java

```diff
@@ -0,0 +1,29 @@
+package org.cbioportal.model;
+
+public class StructuralVariantIdentifier {
```

Comment:
Ik denk dat dit model ook de mogelijkheid moet geven om alle waarden voor een positie (bv gene1), dan wel _null_ gene voor een positie weer te geven. Kijk eens naar de Select class.

Reply:
Added in Renamed in [4c93d75](https://github.com/cBioPortal/cbioportal/commit/4c93d75879595db2e4add0ff8452a5647b540756)